### ISSUE TITLE
when to stop pretraining generator

### ISSUE THREAD

**Issue by jeicy07** [ZhenYangIACAS/NMT_GAN#9]
Hi, when pretraining the generator after 200 epochs, the accuracy almost reaches 1, and loss is extremely small. However, when pretraining discriminator, the accuracy keeps 0.5 from the beginning to the end. I guess the generator is so powerful that the negative samples generated by it are very close to the positive samples. Then the gan training can't move on under this circumstance. So I wonder when should I stop pretraining generator? (at about 70% accuracy or something else?) Thanks.

**Comment by ZhenYangIACAS** [comment]
@jeicy07 You mean the accuracy of your generator reaches near 1?  How large is your training data size? In our experiments, I train the generator as better as possible. However, the accuracy never reaches around 1.

**Comment by jeicy07** [comment]
I use 10w data first to try running whole process. Now I start using 300w data. After 13 epochs, the accuracy keeps at 0.5. Is that normal?

**Comment by ZhenYangIACAS** [comment]
Haven't you ever trained the NMT model?  I think that you should test the BLEU score of your model not just the accuracy.  It is normal for me that the NMT model achieves accuracy around 0.7.

**Comment by jeicy07** [comment]
It's exactly my first time to train an NMT model. Maybe there's something wrong with my data cuz the BLEU score is only 0.09 on evaluation set. Thanks!

**Comment by ZhenYangIACAS** [comment]
@jeicy07 That is all right. Check your data first and you will find something wrong.
